package.json

```diff
@@ -23,8 +24,10 @@
     "lodash": "^4.17.4",
     "marked": "^0.3.6",
     "md-2-json": "^1.0.2",
+    "moment": "^2.18.1",
     "normalize-path": "^2.1.1",
     "pify": "^3.0.0",
+    "request-promise": "^4.2.1",
```

Comment:
You can use `promisify(request)` and use the regular `request` module. That's what we are doing in the [release script](https://github.com/sonarwhal/sonar/blob/7b9d9cf4a017f75e513c48143520ae713af6842e/scripts/release.ts#L98)